testcontainers/mysql.py

```diff
@@ -35,7 +35,7 @@ class MySqlContainer(DbContainer):
     """
 
     def __init__(self, image="mysql:latest", **kwargs):
-        super(MySqlContainer, self).__init__(image)
+        super(MySqlContainer, self).__init__(image, **kwargs)
```

Comment:
It looks like, if `MYSQL_USER` or a similar variable is provided, these would also be passed up the inheritance chain.

Reply:
Changed the constructor parameter list so that `MYSQL_*` are passed as separate arguments and not included in `kwargs`: https://github.com/testcontainers/testcontainers-python/pull/252/commits/322e55f1cad1c181e593ecb03d8bc40be0fefe9e
Please let me know what you think!